packages/@webex/plugin-meetings/test/unit/spec/breakouts/breakout.ts

```diff
@@ -40,43 +41,22 @@ describe('plugin-meetings', () => {
 
     describe('#join', () => {
       it('makes the request as expected', async () => {
-        Metrics.postEvent = sinon.stub();
+        uuid.v4 = sinon.stub().returns('breakoutMoveId');
         const result = await breakout.join();
         assert.calledOnceWithExactly(webex.request, {
           method: 'POST',
           uri: 'url/move',
           body: {
+            breakoutMoveId: 'breakoutMoveId',
+            deviceUrl: undefined,
             groupId: 'groupId',
             sessionId: 'sessionId',
           },
         });
 
         assert.equal(result, 'REQUEST_RETURN_VALUE');
       });
-      it('send metrics as expected', async () => {
```

Comment:
issue: You still need these tests, however instead of testing that the exact metric is fired, you can now stub the breakoutEvent methods and check that they are called as expected

Reply:
restore these tests